tools/interactive/interactivetool_bam_iobio.xml

```diff
@@ -1,46 +1,42 @@
-<tool id="interactive_tool_bam_iobio" tool_type="interactive" name="BAM (iobio) Visualisation" version="0.1">
+<tool id="interactive_tool_bam_iobio" tool_type="interactive" name="bam.iobio visualisation" version="0.2">
```

Comment:
```suggestion
<tool id="interactive_tool_bam_iobio" tool_type="interactive" name="bam.iobio visualisation" version="0.3.0">
```
That syncs it with the iobio version and follows the versioning scheme we've been recommending in the IUC